src/main/java/com/google/devtools/build/lib/analysis/config/BuildConfiguration.java

```diff
@@ -864,6 +864,11 @@ public String getHostCpu() {
     return options.hostCpu;
   }
 
+  public static boolean shouldCreateRunfilesTree(CoreOptions options) {
+    return options.buildRunfilesManifests && options.buildRunfiles;
+  }
+
+  // TODO(buchgr): https://github.com/bazelbuild/bazel/issues/9248
```

Comment:
What's the TODO about? Please note the desired action and its condition.

Reply:
done (inside google)